spec/solr_queries/supplejack_api/more_like_this_search_spec.rb

```diff
@@ -10,7 +10,7 @@ module SupplejackApi
       Sunspot.session = SunspotMatchers::SunspotSessionSpy.new(Sunspot.session)
       @session = Sunspot.session
 
-      # allow(MoreLikeThisSearch).to receive(:role_collection_restrictions) { [] }
+      # allow(MoreLikeThisSearch).to receive(:role_collection_exclusions) { [] }
```

Comment:
**question:** This is commented out. Maybe we can remove this one?

Reply:
done 👍